libsplinter/src/peer/connector.rs

```diff
@@ -60,17 +60,17 @@ impl PeerManagerConnector {
         PeerManagerConnector { sender }
     }
 
-    /// Request that a peer is added to the PeerManager. If a peer already exists, the peer's ref
-    /// count will be incremented
+    /// Requests that a peer is added to the `PeerManager`. If a peer already exists, the peer's
+    /// ref count will be incremented
```

Comment:
Best practice is to spell out "reference" unless the full word would cause confusion.